text/0000-finding-a-balance.md

```diff
@@ -0,0 +1,200 @@
+- Feature Name: balanced-structure
+- Start Date: 2018-04-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a new error style that strikes a better balance between structured output and human readability.
+
+# Motivation
+[motivation]: #motivation
+
+The current `error-style` option of `rustc` provides three styles of error: `human`, `json` or `short`. When using `human`, the compiler will print errors like this:
```

Comment:
You talk about "rustc" and "compiler" here, but https://github.com/rust-lang/cargo/pull/5273#issuecomment-377795660 mentioned this RFC in the context of Cargo. Can you clarify if this RFC is to only apply to rustc output of if Cargo's error should also support this style?